pom.xml

```diff
@@ -135,7 +135,7 @@
     <dependency>
       <groupId>${project.groupId}</groupId>
       <artifactId>ice4j</artifactId>
-      <version>3.0-1-ge854e4e</version>
+      <version>3.0-10-g982e782</version>
```

Comment:
Any other changes in this version bump we should be worried about?

Reply:
Reviewing them I think they're all desirable or harmless.